xml/System.Web.Security.AntiXss/AntiXssEncoder.xml

```diff
@@ -169,10 +169,10 @@
 |[C0 Controls and Basic Latin](https://www.unicode.org/charts/PDF/U0000.pdf)|?|Question mark|  
 |[C0 Controls and Basic Latin](https://www.unicode.org/charts/PDF/U0000.pdf)|@|Commercial at-sign|  
 |[C0 Controls and Basic Latin](https://www.unicode.org/charts/PDF/U0000.pdf)|[ ]|Square brackets|  
-|[C0 Controls and Basic Latin](https://www.unicode.org/charts/PDF/U0000.pdf)|\|Backslash|  
+|[C0 Controls and Basic Latin](https://www.unicode.org/charts/PDF/U0000.pdf)|\\|Backslash|  
```

Comment:
do we also need to escape the other characters here like the @ on line 170 and the * on line 160?